pkgs/tools/misc/tdfgo/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGo118Module, fetchFromGitHub }:
+
+buildGo118Module rec {
```

```suggestion
buildGoModule rec {
```
That is the default go version, also you are missing a top-level entry.

---

I tried this locally and it failed to build.

---

```
error: builder for '/nix/store/nb98rqizzsglyfizf8ivskd3mnxbv2qh-tdfgo-unstable-2022-08-25.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/6gld9jashi8ggkin3y9nxhq3hf8p3axl-user-environment.drv' failed to build
error: 1 dependencies of derivation '/nix/store/yap7vsg676jvnx2r4zc265gnqdzmq34q-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/v65hf4895x3sxcn7q8rb9pgz996q09iz-nixos-system-hyperion-22.05.2134.e43cf174846.drv' failed to build
```

---

You posted the chain of failed builds but not the log of tdfgo and without that we can't help you.

I am assuming you are trying this on nixos 22.11. For contributions please build the package on a checkout of nixpkgs master and test the package from there.